core/src/main/java/dk/alexandra/fresco/lib/generic/BroadcastComputation.java

```diff
@@ -17,13 +16,19 @@
     Computation<List<byte[]>, BuilderT> {
 
   private final List<byte[]> input;
+  private final boolean runValidation;
 
-  BroadcastComputation(List<byte[]> input) {
+  public BroadcastComputation(List<byte[]> input, boolean runValidation) {
```

Comment:
I think it is reasonable to document this class and the constructor - especially when it it resonable to use one or the other